### ISSUE TITLE
Quadvees in Vehicle Mode PSR Issues

### ISSUE THREAD

**Issue by DevianID** [MegaMek/megamek#5752]
Quadvees have lots of little things

### Environment
Tested on 49.19.1

### Description
1. Quadvee's are getting a -1 'dedicated pilot' bonus to PSRs.  In interstellar operations, they get no such 'dedicated pilot' bonus.  In vehicle mode, they dont take PSRs from damage though, which is working correctly.  Tripods get a -1 'dedicated pilot' bonus 'when operating a tripod' which is probably where the quadvee bonus came from, but this -1 bonus only applies to tripods, not quadvees.

2. Quadvee's wheel bonus is not supposed to be effected by things like superchargers/tsm, per forum ruling on the bg battletech boards.  However the Notos prime with wheels can supercharge for 12 in vehicle mode.

3. Wheel hits should reduce the wheel'd MP by 1/4th per crit, like the track equipment in TW.  In testing, a wheel crit did not reduce the notos's speed down from 6/9.  This is confirmed on the forums as well.

4. Physicals kicks against a quadvee *in vehicle mode* should be on the *punch table 'from above', for being a mech 1 level less then another mech.  In testing, they are on the full body table like if the quadvee was prone.  "If subject to physical attacks from other ’Mechs, the QuadVee is treated as a ’Mech that stands 1 level lower than its present level (thus a QuadVee being attacked by a BattleMech on the same level is treated as if it is actually standing one level below 
its attacker)."

6. The BV for speed increase from wheels isnt being applied to quadvees, so the 6/9 Notos with wheels is only paying an offensive speed factor of 5/8 in the megamek calculations.  But this may require a ruling from the Errata people on the forums.

MOD NOTE: Edit for readability

**Comment by RaozSpaz** [comment]
The physical kicks part Im fairly certain is wrong. Since a QuadVee in mech mode is still 2 tall. Only in Vee mode is it 1 tall.

**Comment by DevianID** [comment]
> The physical kicks part Im fairly certain is wrong. Since a QuadVee in mech mode is still 2 tall. Only in Vee mode is it 1 tall.

I did quote the exact rule above.  They are not exactly 1 tall.  For LOS shooting they count as 1 tall, but for physicals it says they are a mech that is standing 1 level below its attacker.  So, like when a mech is on a level 1 hill next to another mech, you kick or hatchet quadvees on the kick table, same as the mech on a hill versus normal mechs

**Comment by RaozSpaz** [comment]
Yeah no. Only in Vee mode are they 1 tall. In Battlemech mode they use the standard rules of a Quad Mech

**Comment by DevianID** [comment]
> Yeah no. Only in Vee mode are they 1 tall. In Battlemech mode they use the standard rules of a Quad Mech

Again, incorrect.  I literally quoted the rules.  For LOS they are level 1 tall.  For physical attacks, you use the rules I quoted, right from the rulebook.  If your quote is different from mine, please post your rules quote.  Its possible the book changed, though I checked the errata and didnt see anything.

**Comment by RaozSpaz** [comment]
Mech Mode
![Screenshot_20240718_184854_Samsung_Internet](https://github.com/user-attachments/assets/217f159b-e400-48c9-903a-4a1986844828)
Vehicle Mode
![Screenshot_20240718_184916_Samsung_Internet](https://github.com/user-attachments/assets/4f5ff4fc-7831-441e-bffc-3ab66b222fa9)

Both say to utilize the standard four legged mech rules with the following additions. Note that anything relative to being a height 1 unit is not mentioned AT ALL in the battlemech section.

**Comment by DevianID** [comment]
> Both say to utilize the standard four legged mech rules with the following additions. Note that anything relative to being a height 1 unit is not mentioned AT ALL in the battlemech section.
In the pic you posted, it says the same as my book.  Quote"The Quadvee is treated as a mech that stands 1 level lower then its present level"

**Comment by RaozSpaz** [comment]
Okay. You finally clarified that you were specifically talking about in Vehicle mode.

**Comment by DevianID** [comment]
> Okay. You finally clarified that you were specifically talking about in Vehicle mode. Nowhere in your initial post did you make mention this Issue was related solely to the vehicle mode of a QuadVee. I highly suggest editing your title to specifically relate to that.

Seeing as vehicle mode has nothing to do with the title/rest of the issues, I put *in vehicle mode* in the one section that applies to just vehicle mode, for better clarity.

**Comment by HammerGS** [comment]
You should confirm what printing is being referenced to ensure we are on the correct page.

**Comment by neoancient** [comment]
These are largely unrelated issues and should have been opened separately.

#2. Do you have a link for this ruling? I can only find a ruling that MASC and TSM do not apply to vehicle mode (https://bg.battletech.com/forums/index.php?topic=49633.msg1145717#msg1145717), but nothing about superchargers.

#5: Awaiting rules question response. Link to thread: https://bg.battletech.com/forums/index.php?topic=85536.msg2021690#msg2021690

**Comment by DM0000** [comment]
Are the things that needs to be fixed on this the punch table and supercharger?
